pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -452,7 +452,7 @@ self: super: {
   #
   # # Depends on itself for testing
   # doctest-discover = addBuildTool super.doctest-discover
-  #   (if pkgs.buildPlatform != pkgs.hostPlatform
+  #   (if pkgs.buildPlatform != pkgs.stdenv.hostPlatform
```

Comment:
Since I have apparently been summoned here because of this change in a comment, I have to ask: Why only change one side of this inequality?

Reply:
fixed